design/initial-draft.md

```diff
@@ -32,8 +32,8 @@ The initial workload implementation is at the “Proof of Concept” level.
 Currently, there is an active effort to redesign Workload to enable payload support for a much broader spectrum of 
 Kubernetes artifacts. At this time the working concepts replacing Workload are:
 
-*   _KubernetesApplication_
-*   _KubernetesApplicationResource_
+*   _KuberntesApplication_
```

Comment:
The old types are correct I think, and spelled correctly
```
_KubernetesApplication_
_KubernetesApplicationResource_
```
This change from KAR -> KAC happens in multiple places in this doc, is that intentional? in master i still see it as `KubernetesApplicationResource`, so this doc should be in agreement with whichever one is correct.